assets/js/modules/analytics-4/components/dashboard/AudienceSegmentation/AudienceTiles.js

```diff
@@ -0,0 +1,347 @@
+
+	// An array of audience entity names.
+	const configuredAudiences = useSelect( ( select ) =>
+		select( MODULES_ANALYTICS_4 ).getConfiguredAudiences()
+	);
+	const audiencesDimensionFilter = {
+		filter: {
+			fieldName: 'audienceResourceName',
+			inListFilter: {
+				values: configuredAudiences,
+			},
+		},
+	};
+	const audiences = useSelect( ( select ) =>
+		select( MODULES_ANALYTICS_4 ).getAudiences()
+	);
+
+	const dates = useSelect( ( select ) =>
+		select( CORE_USER ).getDateRangeDates( {
+			offsetDays: DATE_RANGE_OFFSET,
+			compare: true,
+		} )
+	);
+
+	const reportOptions = {
+		...dates,
+		dimensions: [ { name: 'audienceResourceName' } ],
+		dimensionFilter: audiencesDimensionFilter,
+		metrics: [
+			{ name: 'totalUsers' },
+			{ name: 'sessionsPerUser' },
+			{ name: 'screenPageViewsPerSession' },
+			{ name: 'screenPageViews' },
+		],
+	};
+
+	const report = useSelect( ( select ) => {
+		return select( MODULES_ANALYTICS_4 ).getReport( reportOptions );
+	} );
+
+	const { rows = [] } = report || {};
+
+	const totalPageviewsReportOptions = {
+		startDate: dates?.startDate,
+		endDate: dates?.endDate,
+		metrics: [ { name: 'screenPageViews' } ],
+	};
+
+	const totalPageviewsReport = useSelect( ( select ) => {
+		return select( MODULES_ANALYTICS_4 ).getReport(
+			totalPageviewsReportOptions
+		);
+	} );
+
+	const totalPageviews =
+		totalPageviewsReport?.totals?.[ 0 ]?.metricValues?.[ 0 ]?.value || 0;
+
+	const topCitiesReportOptions = {
+		startDate: dates?.startDate,
+		endDate: dates?.endDate,
+		dimensions: [ 'city' ],
+		metrics: [ { name: 'totalUsers' } ],
+		orderby: [
+			{
+				metric: {
+					metricName: 'totalUsers',
+				},
+				desc: true,
+			},
+		],
+		limit: 3,
+	};
+
+	const topCitiesReport = useSelect( ( select ) => {
+		return select( MODULES_ANALYTICS_4 ).getReportForAllAudiences( {
+			options: topCitiesReportOptions,
+			audiences,
+		} );
+	} );
+
+	const topContentReportOptions = {
+		startDate: dates?.startDate,
+		endDate: dates?.endDate,
+		dimensions: [ 'pagePath' ],
+		metrics: [ { name: 'screenPageViews' } ],
+		orderby: [ { metric: { metricName: 'screenPageViews' }, desc: true } ],
+		limit: 3,
+	};
+
+	const topContentReport = useSelect( ( select ) => {
+		return select( MODULES_ANALYTICS_4 ).getReportForAllAudiences( {
+			options: topContentReportOptions,
+			audiences,
+		} );
+	} );
+
+	const topContentPageTitlesReportOptions = {
+		startDate: dates?.startDate,
+		endDate: dates?.endDate,
+		dimensions: [ 'pagePath', 'pageTitle' ],
+		metrics: [ { name: 'screenPageViews' } ],
+		orderby: [ { metric: { metricName: 'screenPageViews' }, desc: true } ],
+		limit: 15,
+	};
+
+	const topContentPageTitlesReport = useSelect( ( select ) => {
+		return select( MODULES_ANALYTICS_4 ).getReportForAllAudiences( {
+			options: topContentPageTitlesReportOptions,
+			audiences,
+		} );
+	} );
+
+	return (
+		<Widget className="googlesitekit-widget__audience-tiles">
```

Comment:
It also looks like we should use the `noPadding` prop here so we can remove the corresponding `padding: 0` entry in the SCSS file:

```suggestion
		<Widget className="googlesitekit-widget__audience-tiles" noPadding>
```